tests/d3d12_crosstest.h

```diff
@@ -411,6 +411,8 @@ static inline bool is_depth_clip_enable_supported(ID3D12Device *device)
 static PFN_vkGetInstanceProcAddr pfn_vkGetInstanceProcAddr;
 static bool init_vulkan_loader(void)
 {
+    void *mod;
```

Comment:
What if mod is `HMODULE`? and why does this even compile on mingw?

Reply:
HMODULE == void*

Reply:
you are still declaring mod two times tho in the `_WIN32` path

Reply:
I guess it's not as easy to trigger this case since it should only hit when using vkd3d-utils based Win32 test build.

Reply:
Resolved.